packages/p2p/src/components/modal-manager/modals/daily-limit-modal/daily-limit-modal.tsx

```diff
@@ -68,36 +72,20 @@ const DailyLimitModal = () => {
     };
 
     const getModalFooter = () => {
-        if (my_profile_store.is_loading_modal_open) {
+        if (is_daily_limit_upgrading) {
             return null;
-        } else if (my_profile_store.is_daily_limit_success_modal_open || my_profile_store.is_error_modal_open) {
-            return (
-                <Button
-                    has_effect
-                    text={localize('Ok')}
-                    onClick={() => {
-                        my_profile_store.setIsDailyLimitModalOpen(false);
-                    }}
-                    primary
-                    large
-                />
-            );
+        } else if (is_daily_limit_upgrade_success || is_there_daily_limit_error) {
+            return <Button has_effect text={localize('Ok')} onClick={hideModal} primary large />;
```

Comment:
has as well.. the localize

Reply:
suree, yeah we should get rid of the `text` prop from Button component IMO, and use `Localize`